conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -55,12 +55,9 @@ install:
     - cmd: conda install -c pelson/channel/development --yes --quiet obvious-ci
     {%- for channel in channels.get('sources', []) %}
     - cmd: conda config --add channels {{ channel }}{% endfor %}
-    - cmd: conda info
     - cmd: conda install -n root --quiet --yes conda-build anaconda-client jinja2 setuptools
```

Comment:
It is curious that these lines remain. conda-forge-build-setup should have these as dependencies IMO.